specs/card.proto

```diff
@@ -20,6 +20,10 @@ message NuernbergPassNumberExtension {
   optional uint32 pass_number = 1;
 }
 
+message NuernbergIdExtension {
+  optional uint64 pass_id = 1;
+}
```

Comment:
Drive-By Comment:

AFAIK, we should not add this to the CardInfo: It should only be printed onto the PDF.
The only use case for it is, to apply for the discount of the Deutschlandticket; embedding this into all QR codes just bloats the QR codes imo.